compose/foundation/foundation/src/skikoMain/kotlin/androidx/compose/foundation/text/ContextMenuStrings.kt

```diff
@@ -44,8 +44,12 @@ internal value class Strings(val value: Int) {
 @Composable
 @ReadOnlyComposable
 internal fun getString(string: Strings): String {
+    return getLocalizedString(string)
+}
 
+internal fun getLocalizedString(string: Strings): String {
```

Comment:
Why keep both functions? If you think `getLocalizedString` is better, just rename it...